manager/src/main/resources/db/migration/0.1/V0.1.11__update_bridge.sql

```diff
@@ -0,0 +1,11 @@
+-- add new "organisation_id" column
+alter table BRIDGE
+    add column "organisation_id" varchar(255);
+
+-- update existing BRIDGE "organisation_id" with '15247674' value (the only possible one before this migration)
+update BRIDGE
+set "organisation_id"='15247674';
```

Comment:
Make sure to remove this hardcoded value before merge

Reply:
`organisation_id` is non-null of BRIDGE table. We have to provide default value of preexisting rows in table

Reply:
Yes I see the problem but we probably need to do this migration manually for now or similar: it is not possible to find a value that works for every existing rows and for every environment ( see comment https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/748#discussion_r870783582 )